content/learn/workflow.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Sample Quire Workflow
+weight: 6600
+type: page
+abstract:  "An example of Getty Publication's workflow for Quire books"
+---
+
+Below, we outline the workflow Getty Publications's follows when creating [Quire books](https://www.getty.edu/publications/digital/digitalpubs.html). We understand that many organizations have smaller staffs working on publications, but we hope this documentation of our own internal process can provide insight and serve as a jumping off point for developing your organization's own procedure.
+
+## Project Roles
+
+Author(s) <br/>
+Lead project editor <br/>
+Digital team <br/>
+Copy editor<br/>
+Volume editor(s) <br/>
+Proofreader <br/>
+Production
+
+*Note: The lead project editor and copy editor have a basic understanding of Quire and GitHub, while the digital team is more advanced in their knowledge.*
+
+## Pre-transmittal
+
+-   The **authors** submit their manuscripts as Word files to the **lead project editor.**
+
+-   The document undergoes editorial review and peer review before being sent back to the **authors** for review.
+
+-   At this point the **lead project editor** meets with the **digital team** for a pre-transmittal meeting. Three things are covered:
+
+1. Review any special tagging that needs to be done to the files to ensure a clear conversion from Word document to [Markdown](/documentation/fundamentals/#markdown-basics) file. This helps the text-editing process run smoothly and efficiently.
+
+2. Identify special needs such as the use of videos, pop-ups, IIIF images, etc.
+
+3. Assess the content model of the book to identify unique attributes or places where the pattern of the book diverges from the norm. Quire is built to look for patterns so this is a critical step that will inform the data structuring of the publication and YAML formatting.
```

Comment:
Try indenting these lines in twice so that they appear as a sub list of the bullet point above.

Reply:
For some reason the bullets and numbers are combining.

<img width="501" alt="Screen Shot 2022-02-08 at 12 35 03 PM" src="https://user-images.githubusercontent.com/62075223/153071137-958da2a5-63db-4bc3-9c72-1092c1179a0b.png">

Reply:
Just pushed a CSS change that should fix this